xml/smb_native_os.xml

```diff
@@ -257,30 +263,6 @@
     <param pos="0" name="os.cpe23" value="cpe:/o:microsoft:windows_server_2008:-"/>
   </fingerprint>
 
-  <fingerprint pattern="^Windows Server 2016(?: Technical Preview \d+)? (\w+|\w+ \w+|\w+ \w+ \w+)(?: Evaluation)? (\d+)$">
```

Comment:
These were simply moved lower in the file so that the Win 2008 FPs could be grouped